packages/ui-components/src/components/Chart/Table/getChartTableData.js

```diff
@@ -92,7 +95,15 @@ const processData = viewContent => {
 };
 
 export const getChartTableData = viewContent => {
-  const columns = useMemo(() => processColumns(viewContent), [JSON.stringify(viewContent)]);
+  // Because react-table wants its sort function to be memoized, it needs to live here, outside of
+  // the other useMemo hooks
+  const sortByTimestamp = useMemo(() => (rowA, rowB) => {
+    const rowAMoreRecent = rowA.original.timestamp > rowB.original.timestamp;
+    return rowAMoreRecent ? 1 : -1;
+  });
```

Comment:
Little awkward that we need to memoize it here and thus it has to be passed through as a parameter, but if it can't be helped then so be it 👍

Reply:
Yeah I agree...but they've bolded the bit that says you need to memoize it in the react-table docs, so I reckon we'd better do what they say!